src/NHibernate/SqlCommand/SqlStringBuilder.cs

```diff
@@ -158,10 +158,9 @@ public SqlStringBuilder AddObject(object part)
 		/// </summary>
 		/// <param name="sqlString">The SqlString to add to this SqlStringBuilder</param>
 		/// <returns>This SqlStringBuilder</returns>
-		/// <remarks>This calls the overloaded Add(sqlString, null, null, null, false)</remarks>
 		public SqlStringBuilder Add(SqlString sqlString)
 		{
-			sqlString.Visit(AddingVisitor);
+			sqlString?.Visit(AddingVisitor);
```

Comment:
Additional change for avoiding blowing with a null reference exception when the `sqlString` is null. The usual behavior for string builder being to do nothing on null, doing the same here.